### ISSUE TITLE
dV_cloth_gravity_dq added to force in assignment_setup.h

### ISSUE THREAD

**Issue by abhimadan** [dilevin/CSC2549-a4-cloth-simulation#6]
In `assignment_setup.h`, the potential energy derivative for gravity is added to the force instead of subtracted. Shouldn't it be subtracted instead, since the negative of the potential energy derivative is force?

**Comment by dilevin** [comment]
yes I accidentally implanted the force in the solution. I will update the assignment code
